src/hooks/useAlert.ts

```diff
@@ -0,0 +1,25 @@
+import { useDispatch } from 'react-redux';
+
+import type { AppDispatch } from '../common/store';
+import { clearAlert, setAlert } from '../reducers/AlertSlice';
+import type { AlertType } from '../types/state';
+
+const useAlert = () => {
+  const dispatch: AppDispatch = useDispatch();
+
+  const useClearAlert = () => {
+    dispatch(clearAlert());
+  };
+
+  const useSetAlert = (
+    message: AlertType['message'],
+    severity: AlertType['severity'],
+    title: AlertType['title'] = undefined,
+  ) => {
+    dispatch(setAlert({ message, title, severity }));
+  };
+
+  return { useClearAlert, useSetAlert };
```

Comment:
I find the names a bit confusing: they look like they are hooks themselves, but they actually dispatch an action.
I would use simply `clearAlert` and `setAlert`, or more explicit `dispatchClearAlert` and `dispatchSetAlert`. (but I prefer the first option).

Reply:
I can't use `clearAlert` and `setAlert` as those are the names of the reducer actions, but I can change it to `dispatch` instead of `set`